ddl/column.go

```diff
@@ -223,6 +223,16 @@ func onDropColumn(t *meta.Meta, job *model.Job) (ver int64, _ error) {
 		job.State = model.JobStateCancelled
 		return ver, errors.Trace(err)
 	}
+	if job.IsRollingback() && job.Type == model.ActionDropColumn {
```

Comment:
`job.Type == model.ActionDropColumn` is redundant?

Reply:
Fine, onDropColumn will be called by onAddColumn.

Reply:
Add column will rollback the type of the job to be checked with onDropcolumn